x-pack/plugins/security_solution/public/management/cypress.config.ts

```diff
@@ -10,7 +10,7 @@ import { defineCypressConfig } from '@kbn/cypress-config';
 import { dataLoaders } from './cypress/support/data_loaders';
 
 export default defineCypressConfig({
-  defaultCommandTimeout: 60000,
+  defaultCommandTimeout: 120000,
```

Comment:
Where things timing out?

Reply:
I was thinking they were. I previously needed to adjust some other tests that used united index to 120s but looks like it isn't the case here so going to revert and skip for now.